Xamarin.Forms.Core/EffectiveFlowDirection.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Xamarin.Forms
+{
+	[Flags]
+	public enum EffectiveFlowDirection
+	{
+		LeftToRight = 1 << 0,
+		RightToLeft = 1 << 1,
```

Comment:
so the EffectiveFlowDirection can be both LTR and RTL ? and both Implicit and Explicit ?

if not, you might want

```
[Flags]
public enum EffectiveFlowDirection
{
    LTR = 1 << 0;
    Implicit = 1<< 1;
}
```

and add helpers to check mask and evaluate for the negative values. Also, as LTR is the default, I'd give it the `0` value, and `1` to RTL

Reply:
I realize its a slight bastardization of a Flags enum, but its never used by users (and should probably go in the internals namespace) and removes any ambiguity if a 3rd flow direction or implicit/explict convention comes up.

Reply:
Sam points out EffectiveFlowDirection needs to be public for custom renderers, so no internals namespace

Reply:
"It's never used by users" seriously contradicts the "needs to be public for custom renderers". Please fix as this is public API